### PR TITLE
doc(doc.go) fix hello world code example

### PR DESCRIPTION
**Pull request by Pindar**
- properly implement predicate interface
- add naming to routes.eskip to avoid parsing errors
- add missing imports to hello.go

### CODE REVIEW

File: doc.go [hunk 1]
```diff
@@ -350,9 +352,10 @@ Example, hello.go:
     }
 
 A file containing the routes, routes.eskip:
-
-    Random(.05) -> hello("fish?") -> "https://fish.example.org";
-    * -> hello("world") -> "https://www.example.org"
+    
+    hello:
+        Random(.05) -> hello("fish?") -> "https://fish.example.org";
+        * -> hello("world") -> "https://www.example.org"
```
Comment (aryszka) [inline doc.go]: actually, both routes should have a name. E.g:

```
random:
        Random(.05) -> hello("fish?") -> "https://fish.example.org";
hello:
        * -> hello("world") -> "https://www.example.org"
```

**Comment by aryszka** [comment]
thanks a lot!

File: doc.go (hunk 1 above)
Comment (Pindar) [reply doc.go]: absolutely. I missed it during my copy/paste. 
It's updated now.

**Comment by szuecs** [comment]
:+1:

**Comment by aryszka** [comment]
:+1:

**Pull request merged**